docker-cloud/infrastructure/link-aws.md

```diff
@@ -5,14 +5,9 @@ aliases:
 description: Link your Amazon Web Services account
 keywords:
 - AWS, Cloud, link
-menu:
-  main:
-    parent: infrastructure
-title: Link to Amazon Web Services hosts
+title: Link your Amazon Web Services account
```

Comment:
s/your/an
(I realize we're inheriting this from an nonoptimal original title, but if we're changing it anyway...)